src/components/CurrentUser/CurrentUser.tsx

```diff
@@ -1,12 +1,62 @@
 import React from 'react';
+import { getUserByID } from '../../api';
 import './CurrentUser.scss';
 
-export const CurrentUser: React.FC = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+interface Props {
+  userId: number,
+  clearSelection: () => void,
+}
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+interface State {
+  user: User,
+}
+
+export class CurrentUser extends React.Component<Props, State> {
+  state = {
+    user: {} as User,
+  };
+
+  componentDidMount() {
+    this.loadUserInfo();
+  }
+
+  componentDidUpdate() {
+    if (this.state.user.id !== this.props.userId) {
+      this.loadUserInfo();
+    }
+  }
+
+  async loadUserInfo() {
+    const user: User = await getUserByID(this.props.userId);
+
+    this.setState({
+      user,
+    });
+  }
```

Comment:
If user choose someone whith `id` that dose not exist in `Users` array(2, 182, 8452)  he will see nothing, the error will be visible only in the console, but on the screen nothing changes.